src/v/wasm/parser/parser.cc

```diff
@@ -234,6 +234,34 @@ declaration::global parse_global_type(iobuf_parser_base* parser) {
     return {.valtype = valtype, .is_mutable = bool(mut)};
 }
 
+void skip_global_constexpr(iobuf_parser_base* parser) {
+    auto opcode = parser->consume_type<uint8_t>();
+    // NOLINTBEGIN(cppcoreguidelines-avoid-magic-numbers)
```

Comment:
nitpick: I think I'd side w/ the linter here (e.g. if there's a canonical name), though admittedly the meaning is reasonably encoded in the case bodies.

Reply:
This is consistent with other places in this file - but maybe you're right it's worth fixing the lint instead. Let me take a separate pass at something for this full file.